tests/util.test.ts

```diff
@@ -11,6 +11,21 @@ const testDirSimpleIgnore = 'tests/test-func-ignore';
 const testDirNodeIgnore = 'tests/test-func-ignore-node';
 const name = `zip-${Math.round(Math.random() * 100000)}`;
 describe('Zip', function () {
+  it('raises an error if sourceDir does not exist', async function () {
+    try {
+      await zipDir('/not/a/real/path', path.posix.join(os.tmpdir(), name));
+      throw new Error('Should have throw error');
+    } catch (err) {
+      if (err instanceof Error) {
+        const msg = err.message || '';
+        if (msg.includes('Unable to find')) {
+          return;
+        }
```

Comment:
Could we just do `expect(message).contains('Unable to find')`. This should throw an err if it does not contain the err string?